tests/run/datetime_cimport.pyx

```diff
@@ -40,3 +42,19 @@ def test_timedelta(int days, int seconds, int useconds):
     '''
     val = timedelta(days, seconds, useconds)
     return val
+
+def test_timezone(int days, int seconds, int useconds, str name):
+    '''
+    >>> val = test_timezone(0, 3600, 0, 'CET')
+    >>> print(val)
+    True
+    '''
+    try:
+        val = timezone_new(timedelta(days, seconds, useconds), name)
+        return repr(val) == "datetime.timezone(datetime.timedelta(seconds=3600), 'CET')"
```

Comment:
We shouldn't rely on specific output from CPython that may change in future versions. We are not testing CPython here but our own integration with it.

Reply:
Yeah I suppose you're right there. This is mostly how the other tests work though (albeit with `__str__`);
```py
def test_timedelta(int days, int seconds, int useconds):
    '''
    >>> val = test_timedelta(30, 0, 0)
    >>> print(val)
    30 days, 0:00:00
    '''
    val = timedelta(days, seconds, useconds)
    return val
```

Reply:
Just tested and using `__str__` on a timezone returns the name, which is not exactly helpful. I am tempted to basically copy the code from `datetime_members` and test the members. But that's not exactly helpful since we already have that.

I think I'll just use this test to make sure it raises an exception as expected.